app/src/main/java/com/cornellappdev/transit/ui/screens/RouteScreen.kt

```diff
@@ -0,0 +1,164 @@
+import androidx.compose.material3.Icon
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Place
+import androidx.compose.material.icons.outlined.Face
+import androidx.compose.material.icons.outlined.KeyboardArrowLeft
+import androidx.compose.material.icons.outlined.Refresh
+import androidx.compose.material3.Divider
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.Text
+import androidx.compose.material3.TopAppBar
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.TextUnit
+import androidx.compose.ui.unit.TextUnitType
+import androidx.compose.ui.unit.dp
+import com.cornellappdev.transit.R
+import com.cornellappdev.transit.ui.components.MenuItem
+import com.cornellappdev.transit.ui.theme.DividerGrey
+import com.cornellappdev.transit.ui.theme.MetadataGrey
+import com.cornellappdev.transit.ui.theme.PrimaryText
+import com.cornellappdev.transit.ui.theme.SecondaryText
+import com.cornellappdev.transit.ui.theme.sfProDisplayFamily
+import com.cornellappdev.transit.ui.viewmodels.HomeViewModel
+import com.google.accompanist.permissions.ExperimentalPermissionsApi
+
+@OptIn(ExperimentalPermissionsApi::class, ExperimentalMaterial3Api::class)
+@Composable
+fun RouteScreen(
+    //homeViewModel: HomeViewModel = hiltViewModel()
+) {
+
+    Column(modifier = Modifier.fillMaxSize()) {
+        //TODO make an AppBarColors class w/ the right colors and correct icon
+        TopAppBar(
+            title = { Text(text = "Route Options", fontFamily = sfProDisplayFamily, fontStyle = FontStyle.Normal)},
+            navigationIcon = {
+                IconButton(onClick = { /*TODO make this button nav back to whatever screen*/ }) {
+                    Icon(
+                        imageVector = Icons.Outlined.KeyboardArrowLeft,
+                        contentDescription = ""
+                    )
+                }
+            }
+
+        )
+
+        Divider(thickness = 1.dp, color = DividerGrey)
```

Comment:
I did not know `Divider` was a thing, that's nice! :-)

Reply:
i was in the jetpack compose documentation trenches